debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/OptInfo.java

```diff
@@ -220,15 +221,15 @@ private static String getOption(J9ROMClassPointer romClass, long option) throws
 	}
 
 	private static VoidPointer getStructure(J9ROMClassPointer romClass, long option) throws CorruptDataException {
-		SelfRelativePointer ptr = getSRPPtr(romClass.optionalInfo(), romClass.optionalFlags(), option);
+		SelfRelativePointer ptr = getSRPPtr(J9ROMClassHelper.optionalInfo(romClass), romClass.optionalFlags(), option);
```

Comment:
What happens if there is code outside this project using optionalInfo(), cpShapeDescription()? jdmpview supports adding debug extensions at runtime via the plugins command. There are many extensions that exist outside of this project.

Reply:
I don't think there's a way to avoid that problem - it's a consequence of the behavior of ddrgen (being unable to distinguish among `U32Pointer`, `U64Pointer` and `UDATAPointer`).

Reply:
What will happen?

Reply:
Keith created https://github.com/eclipse/openj9/issues/3131 to deal with this.